dubbo-common/src/main/java/org/apache/dubbo/config/AbstractInterfaceConfig.java

```diff
@@ -441,7 +441,7 @@
     }
 
     protected MethodConfig getMethodByName(String name) {
-        if (methods != null && methods.size() > 0) {
+        if (methods != null && !methods.isEmpty()) {
```

Comment:
**Suggestion**: Using '!methods.isEmpty()' instead of 'methods.size() > 0' for checking if the list is not empty is more readable and idiomatic. <br> **Code Suggestion**: 
 ```
 The change correctly implements the suggested improvement.
 ```